lib/countly_flutter.dart

```diff
@@ -1377,23 +1385,16 @@ class Countly {
   }
 
   /// Enable campaign attribution reporting to Countly.
-  /// For iOS use 'recordAttributionID' instead of 'enableAttribution'
-  /// Should be call before Countly init
+  @Deprecated('Use recordIndirectAttribution instead')
   static Future<String?> enableAttribution() async {
-    List<String> args = [];
-    final String? result = await _channel.invokeMethod(
-        'enableAttribution', <String, dynamic>{'data': json.encode(args)});
-    log(result);
-    return result;
+    String error = 'enableAttribution is deprecated, use recordIndirectAttribution instead';
+    log(error);
+    return 'Error : $error';
   }
 
   /// set attribution Id for campaign attribution reporting.
-  /// Currently implemented for iOS only
-  /// For Android just call the enableAttribution to enable campaign attribution.
+  @Deprecated('Use recordIndirectAttribution instead')
   static Future<String?> recordAttributionID(String attributionID) async {
-    if (!Platform.isIOS) {
```

Comment:
this function should print public log for it being called

it should also print a deprecation warning